partiql-ast/src/main/java/org/partiql/ast/DataType.java

```diff
@@ -29,22 +31,17 @@ public final class DataType extends AstEnum {
     @EqualsAndHashCode(callSuper = false)
     public static class StructField extends AstNode {
         @NotNull
-        @Getter
```

Comment:
(self-review) potential issue w/ the `@Getter` (and also `@Builder`) annotation is that they are not included in the generated Javadoc. So in this PR, I ended up using [delombok](https://projectlombok.org/features/delombok) to add the hand-written getters. Some of the getters did have some useful javadoc (especially related to boolean parameters), so including it in the javadoc can be helpful for consumers.

We could apply the same in the future to `@Builder`s if there's a need.